web/src/features/fbaCalculator/components/map/FBAMap.tsx

```diff
@@ -86,6 +107,9 @@ const FBAMap = (props: FBAMapProps) => {
 
     const mapObject = new ol.Map(options)
     mapObject.setTarget(mapRef.current)
+
+    const extent = mapObject.getView().calculateExtent(mapObject.getSize())
```

Comment:
For the dumb - can you please add a link to documentation, or explain what's going on here? I'm sure it super clear if you now gis stuff and this library - but - like - what does mapObject.getSize do? Why is the extent being calculated before buildBCTileLayer ?